plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPlugin.kt

```diff
@@ -307,6 +311,7 @@ class SentryPlugin : Plugin<Project> {
     companion object {
         const val SENTRY_ORG_PARAMETER = "sentryOrg"
         const val SENTRY_PROJECT_PARAMETER = "sentryProject"
+        internal const val SENTRY_SDK_VERSION = "5.6.1"
```

Comment:
yep, this line has to be updated to bump the SDK. I introduce a configuration option called `sentryVersion` which always can be overridden by the users, if they wanna use a version different from the one here.

Alternatively, they can also just add a direct dependency to the SDK and let Gradle resolve it to the latest version. There are many ways this can be configured :)

Reply:
I see. Worth noting that having many ways can possible introduce confusion though. Like "I'm setting version X but it's breaking" And after digging we found out version Y was being set elsewhere which took precedence so worth documenting things thoroughly.